ios/BT/ExposureManager.swift

```diff
@@ -5,242 +5,254 @@ import UserNotifications
 
 @objc(ExposureManager)
 final class ExposureManager: NSObject {
-
-    @objc static let shared = ExposureManager()
-
-    let manager = ENManager()
-
-    var detectingExposures = false
-
-    override init() {
-        super.init()
-        manager.activate { _ in
-            // Ensure exposure notifications are enabled if the app is authorized. The app
-            // could get into a state where it is authorized, but exposure
-            // notifications are not enabled,  if the user initially denied Exposure Notifications
-            // during onboarding, but then flipped on the "COVID-19 Exposure Notifications" switch
-            // in Settings.
-            if ENManager.authorizationStatus == .authorized && !self.manager.exposureNotificationEnabled {
-                self.manager.setExposureNotificationEnabled(true) { _ in
-                    // No error handling for attempts to enable on launch
-                }
-            }
+  
```

Comment:
are these changes from a rebase? or are did we also need to update the native layer for this functionality?

Reply:
We did need to change this, specifically requestExposureNotificationAuthorization.

I think all these changes had something to do with autoformatting. Right @mattThousand??